toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -297,6 +297,10 @@ func addOrUpdateUser(user imagecustomizerapi.User, baseConfigPath string, imageC
 	}
 
 	// Set user's SSH keys.
+	for i, _ := range user.SSHPubKeyPaths {
+		user.SSHPubKeyPaths[i] = filepath.Join(baseConfigPath, user.SSHPubKeyPaths[i])
```

Comment:
- is the result an absolute path?
- what if the user specifies full paths for the ssh key files in the config? Maybe we can check if they are full path, and if not, prepend the baseConfigPath?

Reply:
1) yes
2) good point, updated.